src/components/NavBar/index.jsx

```diff
@@ -28,7 +28,7 @@ const NavBar = () => {
         <NavLink to="/profile" className="nav-link">
           {t("homePage.navbar.profile")}
         </NavLink>
-        <Button variant="outline-info" className="shadow-none">
+        <Button href="/signin" variant="outline-info" className="shadow-none">
```

Comment:
lets put the button inside a NavLink Component and put a ``` to="/signin ```, please check how the signup is done and do the same

Reply:
If we put inside a navlink then the alignment change, why don't we use history push url?

Reply:
@fadime-ozdemir this works too